pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -275,6 +275,7 @@ self: super: {
   dlist = dontCheck super.dlist;
   docopt = dontCheck super.docopt;                      # http://hydra.cryp.to/build/499172/log/raw
   dom-selector = dontCheck super.dom-selector;          # http://hydra.cryp.to/build/497670/log/raw
+  dotenv = dontCheck super.dotenv;
```

Comment:
This should be fixed with a more recent version of `dotenv`, right?  Could you add a comment here stating that?

Reply:
That comment is ok?

Reply:
Thanks, very easy to understand!